README.md

```diff
@@ -2,23 +2,42 @@
 
 [![Build Status](https://travis-ci.org/allegro/ralph-cli.svg?branch=develop)](https://travis-ci.org/allegro/ralph-cli)
 [![Coverage Status](https://coveralls.io/repos/github/allegro/ralph-cli/badge.svg?branch=develop)](https://coveralls.io/github/allegro/ralph-cli?branch=develop)
+[![Documentation Status](https://readthedocs.org/projects/ralph-cli/badge/?version=latest)](http://ralph-cli.readthedocs.io/en/latest/?badge=latest)
 
-A command-line interface for Ralph.
+`ralph-cli` is a command-line interface for [Ralph][ralph]. At this moment, its
```

Comment:
imo we should not put that kind of "negative" info in first sentence of the readme (like "this is ralph-cli, cli interface to ralph, but it has nothing to offer for you"). I'd put emphasis on general idea of this project in first paragraph. Then, in second one, we could describe idea of scan "subcommand" briefly and mention about features-working-right-now. @xor-xor @vi4m what do you think?